docs/developer-docs/functionality/bitcoin/bitcoin.md

```diff
@@ -0,0 +1,219 @@
+
+## Development, Pre-Production, and Production Stage
+
+The Bitcoin functionality including threshold ECDSA is available in all stages required for the development life cycle on the IC:
+
+-   The *SDK* for local development of canisters;
+
+-   IC mainnet support as the pre-production environment for final testing on *Bitcoin testnet*;
+
+-   IC mainnet support as the production environment for the release using *Bitcoin mainnet*.
+
+### Local SDK
+
+In the typical canister development workflow, canisters on the IC are compiled and run in the local environment using the Motoko or Rust canister SDK during their development. Thus, the SDK is the first stage of the development workflow. The SDK has been enabled to support both the Bitcoin integration and threshold ECDSA management canister APIs.
+
+In contrast to the mainnet deployments of the feature, which integrate with Bitcoin testnet and Bitcoin mainnet, respectively, the SDK integrates with a locally-running bitcoind node in regression testing (regtest) mode. Using bitcoind in regtest mode is the preferred way of Bitcoin development. To facilitate our developers as best as possible, we integrated the SDK with bitcoind in regtest mode to bring the best Bitcoin development experience to the IC. Both development and automated testing of smart contracts are first done in the local environment.
+
+The Bitcoin adapter of the single replica running the local SDK environment connects to the local bitcoind node instead of multiple nodes of Bitcoin testnet or mainnet.
+
+### Bitcoin testnet on IC mainnet
+
+Once a smart contract is ready for acceptance testing, it is deployed on IC mainnet (recall, there is no public IC testnet), still using the Bitcoin API set to connect to the Bitcoin testnet. This is the same setting used in local development. This setup is used to perform the acceptance testing of the dApp, using the Bitcoin testnet, i.e., no real value is at stake.
+
+### Bitcoin mainnet on IC mainnet
+
+The final stage of development of a Bitcoin smart contract is its deployment on IC mainnet with the Bitcoin API set to use Bitcoin mainnet. This is the final production environment for the smart contract. This is not yet available, but will be activated with the GA release.
+
+## A Simple Example Canister -- Walkthrough
+
+We will next show how to implement a simple example smart contract using the Bitcoin integration feature. Everything relevant will be explained, from installing the SDK, local development, to deploying the canister on IC mainnet and using the Bitcoin testnet integration.
+
+The following walkthrough uses a minimal example canister that uses all the APIs of the Bitcoin and threshold ECDSA functionality. This code is publicly available on GitHub (link) and can serve as the basis for your own Bitcoin integration project.
```

Comment:
which link will this be on? I'm ok merging this even if the link is not quite live yet.